sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleController.java

```diff
@@ -143,22 +162,19 @@ public void onDiscard() {
 		final CommitDiscardWrapperComponent<SampleEditForm> editView = new CommitDiscardWrapperComponent<SampleEditForm>(form,
 				UserProvider.getCurrent().hasUserRight(UserRight.SAMPLE_EDIT), form.getFieldGroup());
 
-		editView.addCommitListener(new CommitListener() {
-			@Override
-			public void onCommit() {
-				if (!form.getFieldGroup().isModified()) {
-					SampleDto dto = form.getValue();
-					SampleDto originalDto = FacadeProvider.getSampleFacade().getSampleByUuid(dto.getUuid());
-					FacadeProvider.getSampleFacade().saveSample(dto);
-					SormasUI.refreshView();
-
-					if (dto.getSpecimenCondition() != originalDto.getSpecimenCondition() &&
-							dto.getSpecimenCondition() == SpecimenCondition.NOT_ADEQUATE &&
-							UserProvider.getCurrent().hasUserRight(UserRight.TASK_CREATE)) {
-						requestSampleCollectionTaskCreation(dto, form);
-					} else {
-						Notification.show(I18nProperties.getString(Strings.messageSampleSaved), Type.TRAY_NOTIFICATION);
-					}
+		editView.addCommitListener(() -> {
+			if (!form.getFieldGroup().isModified()) {
+				SampleDto dto1 = form.getValue();
```

Comment:
Not a good naming pattern; "changedDto" or "updatedDto" might be an appropriate name, but just adding a number to it in a place where the code logic is actually relevant makes it hard to read the code.